<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>promotionDialog</class>
 <widget class="QDialog" name="promotionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Promotion</string>
   </property>
  </widget>
  <widget class="QLabel" name="queenPic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rookPic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="knightPic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bishopPic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="queenButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Queen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rookButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rook</string>
   </property>
  </widget>
  <widget class="QPushButton" name="knightButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Knight</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bishopButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>113</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bishop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
